### ISSUE TITLE
Future grenades void people forever

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#34156]
(WEB REPORT BY: sacredatom REMOTE: 206.221.180.138:7777)
# Revision
a50eb832e5fa0f538208d061cd267c94696f07b9

# Description
unsure how, it worked fine when I tested it but two players got sent to a void I had to manually pull them out of due to getting caught in nades, didnt even appear on player panel
# Steps to Reproduce
no idea get future naded
# What you Expected
supposed to pop back into reality after 10 seconds
# What Actually Happened
VOIDED

**Comment by Exxion** [comment]
I think it's 30 seconds, not 10, but I assume you waited longer than either

**Comment by SacredAtom** [comment]
Yeah it was several minutes, one guy said he could reply to pda messages but was too braindamaged to actual do anything

**Comment by Eneocho** [comment]
There's some issues if you stack time effects simultaneously, like being able to drop stuff into nullspace if you manage to time both a send to future and sent to past correctly (the interactions return while being sent to the past, so you can drop whatever you held, which stays in nullspace).
